spec/models/initiative_spec.rb

```diff
@@ -0,0 +1,75 @@
+# == Schema Information
+#
+# Table name: initiatives
+#
+#  id          :integer          not null, primary key
+#  description :string
+#  startdate   :date
+#  status      :integer
+#  title       :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  parent_id   :integer
+#
+# Indexes
+#
+#  index_initiatives_on_parent_id  (parent_id)
+#
+# Foreign Keys
+#
+#  parent_id  (parent_id => initiatives.id)
+#
+require 'rails_helper'
+
+RSpec.describe Initiative, type: :model do
+  context 'in its entities,' do
+      it 'may have helpers' do
```

Comment:
Some inconsistent tabs here. this should be indented with 4 spaces

Reply:
As a more relevant comment you should never tests may things. That is not very specific and usually is a poor behavior descriptor. Saying that I "may" have something only means that I can have it or can not have it so it shouldn't be a test because the code has nothing to enforce. If the object is present is cool if the object is absent then is also cool